src/DssSpell.t.sol

```diff
@@ -311,30 +348,30 @@ contract DssSpellTest is DssSpellTestBase {
     //     assertEq(dai.balanceOf(DUX_WALLET) - prevDaiDUX, amountDaiDUX);
     // }
 
-    function testAAVEDirectBarChange() public {
-        DirectDepositLike join = DirectDepositLike(addr.addr("MCD_JOIN_DIRECT_AAVEV2_DAI"));
-        assertEq(join.bar(), 4 * 10**27 / 100);
+    // function testAAVEDirectBarChange() public {
+    //    DirectDepositLike join = DirectDepositLike(addr.addr("MCD_JOIN_DIRECT_AAVEV2_DAI"));
+    //    assertEq(join.bar(), 4 * 10**27 / 100);
 
-        vote(address(spell));
-        scheduleWaitAndCast(address(spell));
-        assertTrue(spell.done());
+    //    vote(address(spell));
+    //    scheduleWaitAndCast(address(spell));
+    //    assertTrue(spell.done());
 
-        assertEq(join.bar(), 3.9 * 10**27 / 100);
-    }
+    //    assertEq(join.bar(), 3.9 * 10**27 / 100);
+    // }
 
-    function testSendFundsFromPauseToVow() public {
-        vote(address(spell));
-        spell.schedule();
-        hevm.warp(spell.nextCastTime());
+    // function testSendFundsFromPauseToVow() public {
+    //    vote(address(spell));
+    //    spell.schedule();
+    //    hevm.warp(spell.nextCastTime());
 
-        jug.drip("GUNIV3DAIUSDC1-A"); // So we do not affect surplus when executing the spell
-        assertEq(dai.balanceOf(address(pauseProxy)), 218_059.1 * 10**18);
-        uint256 prevDaiVowBalance = vat.dai(address(vow));
+    //    jug.drip("GUNIV3DAIUSDC1-A"); // So we do not affect surplus when executing the spell
+    //    assertEq(dai.balanceOf(address(pauseProxy)), 218_059.1 * 10**18);
+    //    uint256 prevDaiVowBalance = vat.dai(address(vow));
 
-        spell.cast();
-        assertTrue(spell.done());
+    //    spell.cast();
+    //    assertTrue(spell.done());
 
-        assertEq(dai.balanceOf(address(pauseProxy)), 0);
-        assertEq(vat.dai(address(vow)), prevDaiVowBalance + 218_059.1 * 10**45);
-    }
+    //    assertEq(dai.balanceOf(address(pauseProxy)), 0);
+    //    assertEq(vat.dai(address(vow)), prevDaiVowBalance + 218_059.1 * 10**45);
```

Comment:
I'd remove these two tests are won't be used often.

Reply:
Done in 716865f.